libs/web/components/src/ProposalNewForm.tsx

```diff
@@ -24,7 +24,7 @@ export const ProposalNewForm: FC<
 
 	useEffect(() => {
 		ref.current?.focus();
-	}, []);
+	}, [ref]);
```

Comment:
Does your IDE complain about this? As `ref` should be stable and there is no need to put in as dependency